lnrpc/routerrpc/router.proto

```diff
@@ -222,167 +222,9 @@ message SendToRouteResponse {
     bytes preimage = 1;
 
     /// The failure message in case the payment failed.
-    Failure failure = 2;
+    lnrpc.Failure failure = 2;
```

Comment:
See discussion in https://github.com/lightningnetwork/lnd/pull/4000#discussion_r385044856

We move them because we add this message to the main rpc `ListPayments` response. It is (build time) breaking, but it is okay because `routerrpc` is still experimental.

Reply:
That would be a breaking change to the main server rpc.

Reply:
Why would it break?

Reply:
Assuming we are talking about 'back then', if we would have moved something like the `Invoice` message into the subserver, the namespace of that message would have changed and possibly cause build issues for users.